dev.geopjr.Hashbrown.yaml

```diff
@@ -0,0 +1,53 @@
+app-id: dev.geopjr.Hashbrown
+runtime: org.gnome.Platform
+runtime-version: "3.38"
+sdk: org.gnome.Sdk
+command: hashbrown
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --filesystem=host:ro
```

Comment:
You should be able to use `GtkFileChooserNative` which transparently uses portal for accessing single files.

Reply:
Thanks!